content/pt/docs/reference/access-authn-authz/authentication.md

```diff
@@ -0,0 +1,892 @@
+ 
+Em contraste a usuários normais, contas de serviço são considerados usuários gerenciados pela API do Kubernetes. Elas estão vinculadas à específicos _namespaces_ e criados automaticamente pelo servidor de API ou manualmente através de chamadas da API. Contas de serviço estão ligadas a um conjunto de credenciais armazenados como `Secrets`, aos quais são montados dentro dos _pods_ assim permitindo que processos internos ao _cluster_  comuniquem-se com a API do Kubernetes.
+ 
+Requisições para a API estão ligadas a um usuário normal, conta de serviço ou serão tratadas como requisições anônimas (#anonymous-requests). Isto significa que cada processo dentro ou fora do _cluster_, desde um usuário humano entrando `kubectl` de uma estação de trabalho, a `kubelets` rodando nos nós, a membros do painel de controle devem autenticar-se ao realizarem suas requisições para o servidor API ou serão tratados como usuário anônimo.
+ 
+## Estratégias de autenticação
+ 
+Kubernetes usa certificados de clientes, _bearer Token_, um proxy realizando autenticação, ou uma autenticação básica HTTP para autenticar requisições para o servidor de API através de plugins. Como requisições HTTP são feitas para o servidor de API, plugins tentam associar os seguintes atributos junto a requisição:
+ 
+* Username: um valor (String) que identifica o usuário final. Valores comuns podem ser `kube-admin` ou `jane@example.com`
+* UID: um valor (String) que identifica o usuário final e tenta ser mais consistente e único do que username.
+* Groups: Um conjunto de valores em que cada item indica a associação de um usuário há uma coleção lógica de usuários. Valores comuns podem ser `system:masters` ou `devops-team`.
+* Campos extras: um mapa para uma lista que armazena informações adicionais em que autorizadores podem achar útil.
+ 
+Todos os valores são opacos para o sistema de autenticação e somente trazem significância quando interpretados por um autorizador (/docs/reference/access-authn-authz/authorization/).
+ 
+É possível habilitar múltiplos métodos de autenticação. Deve-se normalmente usar pelo menos dois métodos:
+ 
+- _Tokens_ para contas de serviço;
+- Pelo menos um outro método de autenticação para usuários.
+ 
+Quando múltiplos módulos de autenticação estão habilitados, o primeiro módulo a autenticar com sucesso uma requisição termina o fluxo de avaliação da mesma.
+ 
+O servidor de API não garante a ordem em que os autenticadores são processados.
+ 
+O grupo `system:authenticated` é incluído na lista de grupos de todos os usuários autenticados.
+ 
+Integrações com outros protocolos de autenticação, como LDAP, SAML, Kerberos, alternate x509 schemes, etc, podem ser alcançadas utilizando-se de um proxy (#authenticating-proxy) ou webhook (#webhook-token-authentication) de autenticação.
```

Comment:
`proxy (#authenticating-proxy)` is `[proxy](#authenticating-proxy)` and `webhook (#webhook-token-authentication)` is `[webhook](#webhook-token-authentication)`

Reply:
Fixed on commit 9b5815e